From 0eeb804b43596402bd7a5397e46c7d4eb7505074 Mon Sep 17 00:00:00 2001
From: Jayanthi A <jayanthi.annadurai@broadcom.com>
Date: Thu, 19 Sep 2013 15:16:04 -0700
Subject: XLP9XX NAE ucore: Defined class override option in default ucore file

To test xlaui interface on xlp9xx board, modify the default
ucore file to include CLASS_OVERRIDE option
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Nam Ninh <nam.ninh@windriver.com>

diff --git a/arch/mips/boot/dts/xlp9xx-helinux.dts b/arch/mips/boot/dts/xlp9xx-helinux.dts
index d958de5..0ebaf55 100644
--- a/arch/mips/boot/dts/xlp9xx-helinux.dts
+++ b/arch/mips/boot/dts/xlp9xx-helinux.dts
@@ -102,7 +102,7 @@
 			};
 
 			cpu {
-				onlinemask = <0xffff>;
+				onlinemask = <0xffffffff>;
                                 nae-rx-vc = <2>;
                                 nae-fb-vc = <3>;
 				dtre-rx-vc = <2>;
@@ -119,7 +119,7 @@
 				id = <0>;
 			};
 			app-param{
-				onlinemask = <0xffff>;
+				onlinemask = <0xffffffff>;
 				enable-nae-hw-parser = <0>;
 				enable-nae-jumbo = <0>;
 				enable-nae-p2p = <0>;
-- 
1.7.1

